main.go

```diff
@@ -92,6 +94,7 @@ func main() {
 	defaultNamespaces := map[string]cache.Config{
 		operatorNamespace:            {},
 		"openshift-storage-extended": {},
+		"ibm-spectrum-scale":         {},
```

Comment:
I see this PR is only about supporting multiple ns and I believe this specific change be part of another PR if we are making changes for the intermediate bundle. Will there be another PR for this effort?